packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -143,9 +154,29 @@ async fn create_data_type<P: StorePool + Send>(
 )]
```

Comment:
Do the return values of the `body` need to be changed?

Reply:
Yes, that was an oversight, thank you!

Reply:
Fixed in e1439339f7982c8bc1ec2f4999fc5d362260840a